model/src/main/java/org/hibernate/search/benchmark/model/param/RelationshipSize.java

```diff
@@ -3,7 +3,7 @@
 public enum RelationshipSize {
 
 	SMALL( 1, 1, 1, new int[] { 3 }, 1 ),
```

Comment:
As discussed on Zulip, it would be great to change this to use the same algorithm for the small relationship size as for medium/large relationship sizes. We could compensate by reducing the number of companies.

Reply:
Yeah, I'm doing that....